build.gradle

```diff
@@ -4,11 +4,13 @@ plugins {
 
 group = 'org.contikios.cooja'
 
-def javaVersion = 17
+def javaVersion = 20
 
 java {
   toolchain {
     languageVersion = JavaLanguageVersion.of(javaVersion)
+    // ADOPTIUM is broken for Java 20, missing java.security file.
+    vendor = JvmVendorSpec.GRAAL_VM
```

Comment:
The GraalVM in Docker is broken, this line is not required with https://github.com/contiki-ng/contiki-ng/pull/2534.